README.md

```diff
@@ -230,6 +231,20 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
 
   We also have a dev slack channel, please follow this [link](https://join.slack.com/t/ai-habitat/shared_invite/enQtNjY1MzM1NDE4MTk2LTZhMzdmYWMwODZlNjg5MjZiZjExOTBjOTg5MmRiZTVhOWQyNzk0OTMyN2E1ZTEzZTNjMWM0MjBkN2VhMjQxMDI) to get added to the channel.
 
+## WebGL
+
+1. Download the [test scenes](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally to habitat-sim creating habitat-sim/data.
+1. Download and install [emscripten](https://emscripten.org/docs/getting_started/downloads.html) (version 1.38.38 is verified to work)
```

Comment:
Same here, these all can be converted to bunch of commands which users can directly copy paste.

Reply:
I'm hesitant to reproduce the emscripten documentation. For Matterport, we just point to their documentation.